### ISSUE TITLE
empty scrollable datatable with fixed scrollHeight shold have Right, Left and Bottom borders

### ISSUE THREAD

**Issue by mtbadi39** [primefaces/primefaces#898]
we have a scrollable datatable with fixed scrollHeight; and gets data from a bisenes service. When the number of rows is greater then the scrollable zone all is fine. but when the number of rows is lesser, it fill the rest with an empty blank space without left and right borders !! As a workaround i add :

```
.ui-datatable-scrollable-body {
   background-color: #e5e5e5;
}
```

with xhtml as : 

```
            <p:dataTable id="noteDetailId"
                         widgetVar="detailLotWidget"
                         var="sitTransfo" 
                         value="#{serviceAchat.listeSitsTransfos}" 
                         selectionMode="single" 
                         selection="#{serviceAchat.selectedSitTransfo}" 
                         rowKey="#{sitTransfo.transfo}"
                         scrollable="true" scrollHeight="calc(99vh - 476px);"
                         emptyMessage="Pas des transfos dans ce lot !"

                         >
                <f:facet name="header">
                    <span></span> <!-- just for top border -->
                </f:facet>
                <p:ajax event="rowSelect" update="noteDetailActionsForm" />
                <p:ajax event="rowUnselect" update="noteDetailActionsForm" />
                <p:column headerText="Fabricant">
                    <h:outputText value="#{sitTransfo.transfo.fabricant}" />
                </p:column>
                <p:column headerText="Puissance" style="text-align: center;" width="90">
                    <h:outputText value="#{sitTransfo.transfo.puissance}" />
                </p:column>
                <p:column headerText="Numéro">
                    <h:outputText value="#{sitTransfo.transfo.numero}" />
                </p:column>
                <p:column headerText="Année Fab." style="text-align: center;" width="90">
                    <h:outputText value="#{sitTransfo.transfo.anneeDeFabrication}" />
                </p:column>
                <!--p:column headerText="Prix d'Achat" style="text-align: right;">
                    <h:outputText value="#{transfo.prixAchat}" />
                </p:column-->
                <!--p:column headerText="Transfo">
                    <h:outputText value="#{sitTransfo.transfo.fabricant}" />
                </p:column-->
                <!--p:column headerText="Etat" width="60">
                    <h:outputText value="#{sitTransfo.etat}" />
                </p:column-->
                <p:column headerText="Emplacement">
                    <h:outputText value="#{sitTransfo.emplacement}" />
                </p:column>

                <f:facet name="footer">
                    <span></span> <!-- just for bottom border -->
                </f:facet>
            </p:dataTable>
```

![scrollable_datatable](https://cloud.githubusercontent.com/assets/8750684/11324526/91f5fe9a-9134-11e5-859c-fd6931552ac4.png)

**Comment by kukel** [comment]
Duplicate? https://github.com/primefaces/primefaces/issues/958 I personally prefer zebra striped empty rows

**Comment by melloware** [comment]
Duplicate of #958 and  was fixed with #4357
